cnd/src/db.rs

```diff
@@ -254,4 +302,28 @@ mod tests {
         assert_that(&db).is_ok();
         assert_that(&path).exists();
     }
+
+    #[test]
+    fn stability_of_serialization_halight_network() {
```

Comment:
We have a file with tests for these in the db module :)

Reply:
Oh cool, I missed that. Thanks.